baselines/webworker.generated.d.ts

```diff
@@ -1154,32 +1245,32 @@ declare var Request: {
     new(input: Request | string, init?: RequestInit): Request;
 };
 
-interface Response extends Object, Body {
+interface Response extends Body {
     readonly body: ReadableStream | null;
     readonly headers: Headers;
     readonly ok: boolean;
+    readonly redirected: boolean;
     readonly status: number;
     readonly statusText: string;
     readonly type: ResponseType;
     readonly url: string;
-    readonly redirected: boolean;
     clone(): Response;
 }
 
 declare var Response: {
     prototype: Response;
-    new(body?: any, init?: ResponseInit): Response;
-    error: () => Response;
-    redirect: (url: string, status?: number) => Response;
+    new(body?: Blob | Int8Array | Int16Array | Int32Array | Uint8Array | Uint16Array | Uint32Array | Uint8ClampedArray | Float32Array | Float64Array | DataView | ArrayBuffer | FormData | string | null, init?: ResponseInit): Response;
```

Comment:
Isn't there a type alias somewhere we could be using in place of `Int8Array | Int16Array | Int32Array | Uint8Array | Uint16Array | Uint32Array | Uint8ClampedArray | Float32Array | Float64Array` here? I think I remember one getting added awhile back.

Reply:
BufferSoruce. it should still be there.

Reply:
done.